src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -68,7 +69,14 @@
     /**
      * Returns the parameters and corresponding values used in the HTTP request to the test endpoint.
      */
-    protected abstract Map<String, String> getTestParameters();
+    protected abstract Map<String, String> getTestEndpointParameters();
+
+    /**
+     * Returns the JMeter variables used in the body of the HTTP POST request to the test endpoint.
+     */
+    protected List<String> getTestEndpointPostVariables() {
```

Comment:
- I believe the standard term for this is "Request body", so the method name should be `getTestEndpointRequestBody`?
- HTTP request body is usually just `String`, is there a need for this to be a `List`?